docs/Manpage.md

```diff
@@ -2052,6 +2052,24 @@ And for an authenticated HTTP proxy:
 
     export http_proxy=http://$USER:$PASSWORD@$HOST:$PORT
 
+## TERMINOLOGY
```

Comment:
I think ideally we'd have this be in the manpage before we use these terms.

Reply:
Right at the beginning after the `DESCRIPTION` section?